package-lock.json

```diff
@@ -5526,9 +5526,9 @@
       "dev": true
     },
     "@types/node": {
-      "version": "10.17.5",
-      "resolved": "https://registry.npmjs.org/@types/node/-/node-10.17.5.tgz",
-      "integrity": "sha512-RElZIr/7JreF1eY6oD5RF3kpmdcreuQPjg5ri4oQ5g9sq7YWU8HkfB3eH8GwAwxf5OaCh0VPi7r4N/yoTGelrA=="
+      "version": "17.0.38",
```

Comment:
@simmkyu @ltrung , will this `upgrading @types/node from 10.17.5 to 17.0.38` be considered as a breaking change?

Reply:
`protobufjs@6.11.3` requires `@types/node": ">=13.7.0`, previously `protobufjs@6.8.8` requires `@types/node": "^10.1.0`